docs/getting-started.md

```diff
@@ -193,77 +207,82 @@ data will be lost. Keep in mind, that it may take a much longer time from 8V to
-you want to use and on the OS of your host computer, so pick the one that applies:
-
-* __Ethernet over USB__ (just requires the USB cable that comes with the EV3)
-    * [Connecting to the Internet](../tutorials/connecting-to-the-internet-via-usb) tutorial
-* __USB Ethernet dongle__ (as in the kind with an RJ45 connector)
-    * If your network has a DHCP server, this will "just work".
-* __USB Wi-Fi Dongle__
-    * Setting up Wi-Fi Via the Brickman User Interface (hopefully you can figure
-      this out, but it would be nice if someone made a tutorial)
-    * [Setting Up Wi-Fi Using the Command Line](../tutorials/setting-up-wifi-using-the-command-line)
-      (requires another connection type first to get to the command line, but it's way
-      easier to enter your passphrase this way)
-* __Bluetooth__
-    * Note: Bluetooth may not work on the first boot. Please reboot if you see "???"
-      after you power on Bluetooth.
-    * [Connecting to the Internet](../tutorials/connecting-to-the-internet-via-bluetooth) tutorial
+Click on the button below to visit our networking page. There are lots of
+choices here; choose the connection option that best suits you and your
+available hardware.
 
 {% include icon.html type="info" %}
 For Raspberry Pi with no display, you must use the wired Ethernet port
 to connect for the first time. You can set up additional connections using the
 `connmanctl` command.
 {: class="alert alert-info"}
 
-Now that you have a network connection, you should
-[connect to your EV3 with SSH](/docs/tutorials/connecting-to-ev3dev-with-ssh)
-if you haven't done so already.
+<a class="btn btn-md btn-primary" href="/docs/networking" target="_blank"><span class="glyphicon glyphicon-globe"></span> Check out the available networking options</a>
 
 {% include end-section.html %}
 
 {% include begin-section.html %}
 
-## Step 6: Do something awesome
+## Step 6: Connect to the EV3 via SSH
+
+Now that you have a network connection, you should connect to your EV3 with SSH.
+Connecting allows you to verify that the networking option that you configured
+in the previous step is working. Once you have an SSH connection, you will be
+able to securely run terminal commands on the EV3 from your PC. This lets you
+run programs, change settings, and install new programs. Click the button below
+to learn how to connect.
+
+<a class="btn btn-md btn-primary" href="/docs/tutorials/connecting-to-ev3dev-with-ssh" target="_blank"><span class="glyphicon glyphicon-transfer"></span> Learn how to connect with SSH</a>
 
-The brick can run almost all programming languages that any other Linux distro can, so your
-favorite language is probably supported. Language bindings have already been
-written for many languages. **You can learn more about the available libraries
-[here](/docs/programming-languages).**
+To test this out, let's try running a command. Type (or copy-and-paste) the
```

Comment:
should probably say "After you have connected using SSH, in the SSH terminal window..." or something to that effect.